CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+- Related Issues - Please note any related issues, if any.
+- Suggest a Fix - If you have an idea of what's causing the problem (line of code or a commit), please point it out.
+
+## <a name="newfeatures"></a> Requesting Features
+
+You can submit an issue in [GitHub Issues][githubissues]. If you would like to implement it yourself, then consider the scope of the change:
+
+- **Major Change**: If you are interested in a major change, please [get in touch](#getting-help) with the team so we can coordinate our efforts, avoid duplicate work, and ensure your contributions are successfully accepted into the project.
+- **Minor Change**: If the change is small, you can fork the [repository][github] and create a Pull Request with your changes. See [Submitting a Pull Request](#pullrequest) for more information.
+
+## <a name="commiting"></a> Git Commit Guidelines
+
+We follow the `conventional-changelog` format used by the AngularJs team. You can follow the format manually, but we highly recommend you use a CLI wizard such as [Commitizen][commitizen].
+
+In fact, we have configured this repository to work with Commitizen. First, ensure Commitizen CLI is installed via `npm install -g commitizen`. Next, make sure the clone of this repository is installed via `npm install`. You're all set! Now, whenever you want to commit, just stage your changes e.g. `git add .` and commit using `git cz`. Follow the prompts and you're done.
+
+
+### Commit Message Format
+
+Each commit message should consist of up to three lines: **header**, **body**, and **footer**. Additionally, the **header** should be a special format containing the **type**, **scope**, and **subject**.
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The **header** is mandatory, however, the **scope** of the header is optional.
+
+Every line should be no longer than 100 characters.
+
+### Revert
+
+Commits that revert a previous commit should begin with `revert: ` followed by the header of the reverted commit. The body should read `This reverts commit <hash>`, where **hash** is the SHA of the commit you are reverting.
+
+### Type
+
+Must be one of the following:
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+
+### Scope
+
+The scope should be the aspect or subfeature being affected, such as a module, element, or collection. For example: `modules/checkbox`, 'elements/button', etc.
```

Comment:
elements/button does not have proper formatting, it has quotes instead of backticks